src/components/PlayerInfo.vue

```diff
@@ -69,7 +69,11 @@ export default class PlayerInfo extends Vue {
   }
 
   get factionDesc() {
-      return `<b>Ability: </b> ${factionDesc[this.player.faction].ability} </br><b>PI: </b> ${factionDesc[this.player.faction].PI} `;
+      return `<b>Cost:</b> ${factionDesc[this.player.faction].cost} </br> 
```

Comment:
It seems to be the same as in Commands.vue. Maybe make it a single method and use it at both locations?

Reply:
moved to factions